### PR TITLE
Strings velo + regenerated master strings

### PR DESCRIPTION
**Pull request by biodranik**
Duplicates and closes #2929

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -3018,9 +3018,8 @@
 
   [share_bookmarks_email_body]
     tags = android,ios
-    en = Hello!\n\nAttached are my bookmarks from Organic Maps app. Please open them if you have Organic Maps installed. Or, if you don't, download the app for your iOS or Android device by following this link: https://omaps.app/get?kmz\n\nEnjoy traveling with Organic Maps!
-    be = Вітаю!\n\nУ далучаным файле мае закладкі з дататка Organic Maps. Калі ласка, адчыніце іх калі ў вас усталяваны Organic Maps. Калі не, спампуйце дадатак для вашай прылады iOS альбо Andriod па спасылцы: https://omaps.app/get?kmz\n\nПрыемных падарожжаў з Organic Maps!
     en = Hallo!\n\nIm Anhang sind meine Lesezeichen der Organic Maps App. Sie können sie mit Organic Maps öffnen. Wenn Sie die App nicht installiert haben, können Sie sie von https://omaps.app/get?kmz für iOS oder Android herunterladen.\n\nViel Spaß beim Reisen mit Organic Maps!
```
Comment (pastk) [inline data/strings/strings.txt:3021]: Should be `de` maybe? ATM it substitutes the original `en` translation.

**Review by pastk — CHANGES_REQUESTED** [review]

File: data/strings/strings.txt (hunk 1 above)
Comment (pastk) [reply data/strings/strings.txt:3021]: It was broken in #2937

Comment (biodranik) [reply data/strings/strings.txt]: Good catch, thanks! Fixed, PTAL

File: data/categories.txt [hunk 2]
```diff
@@ -8683,7 +8684,7 @@ eu:Bizikleta aparkalekua
 sv:4Cykelparkering
 th:ที่จอดจักรยาน
 tr:4Bisiklet park etme
-uk:4Велостоянка
+uk:4Велостоянка|велопарковка
```
Comment (pastk) [inline data/categories.txt:8687]:
```suggestion
uk:4Велопарковка|велостоянка
```

File: data/categories.txt [hunk 3]
```diff
@@ -8660,6 +8660,7 @@ mr:पाळणाघर|अंगणवाडी|बालवाडी
 amenity-bicycle_parking
 en:4Bicycle Parking
 ru:4Велостоянка|велопарковка
```
Comment (pastk) [inline data/categories.txt:8662]:
```suggestion
ru:4Велопарковка|велостоянка
```

**Review by pastk — APPROVED** [review]

**Comment by biodranik** [comment]
@pastk Fixed, PTAL

**Review by pastk — APPROVED** [review]

**Pull request merged**